src/status_im/ui/screens/discover/events.cljs

```diff
@@ -1,7 +1,7 @@
 (ns status-im.ui.screens.discover.events
   (:require [re-frame.core :refer [after dispatch enrich]]
             [status-im.utils.utils :refer [first-index]]
-            [status-im.utils.handlers :refer [register-handler get-hashtags]]
+            [status-im.utils.handlers :refer [register-handler register-handler-fx get-hashtags]]
```

Comment:
How is that you `:refer` `register-handler-fx` and then use the `handlers/register-handler-fx` ?
In general we are trying to move from `:refer`-ing anything to always explicitly aliasing with `:as`